src/languages/swift.js

```diff
@@ -283,6 +288,143 @@ export default function(hljs) {
   };
   TYPE.contains.push(GENERIC_ARGUMENTS);
 
+  // https://docs.swift.org/swift-book/ReferenceManual/Expressions.html#ID552
+  // Prevents element names from being highlighted as keywords.
+  const TUPLE_ELEMENT_NAME = {
+    begin: lookahead(concat(Swift.identifier, /\s*:/)),
+    end: lookahead(/:/),
+    keywords: "_",
```

Comment:
```suggestion
    keywords: "_|0",
```

And what is end trying to accomplish here?  If you just watch to match the identifier portion why not do it with begin with a lookahead for `:` inside begin?

Reply:
I'm trying to detect the element names in a tuple and consume them so they don't get highlighted as other things (keywords, function params, etc).

Reply:
Sample code?  And see my note above about just using a negative look-ahead for `:`... that solves keywords I think.

Reply:
Fixed the relevance in the next commit.